src/main/resources/db/migration/V1_9__add_unique_constraint.sql

```diff
@@ -0,0 +1,3 @@
+-- Increase length of `authorised_by` column to 60 chars as in NOMIS
+ALTER TABLE non_association
+ADD CONSTRAINT prisoner_association_unq UNIQUE (first_prisoner_number, second_prisoner_number)
```

Comment:
Do we need this unique constraint already? If we allow for (potential) duplicated we don't need to over complicate the merge logic right? (as we can simply replace `oldPrisonerNumber` with `newPrisonerNumber` as you were doing before)

Reply:
i think we're _forced_ into this situation rather than want it! i would expect that the rest of the team would think that closed non-associations remain in the history forever rather than getting re-opened with new details

Reply:
Ok I'll remove ! - Think we need to have a rule about only one open -discuss tomorrow